### ISSUE TITLE
Strange square wave behaviour - bug?

### ISSUE THREAD

**Issue by kasperkamperman** [processing/processing-sound-archive#47]
I was recording a video tutorial where I was using the SqrOsc with an amplitude of 0.5. 

```
squareWave.freq(400);
squareWave.amp(0.5);
```

No problem, however when I restarted working on it I noticed that the SqrOsc hardly made sound at that amplitude, while before I had a nice volume. 

I uninstalled the library, restarted Processing, installed it again. But it keep the same. 

The strange thing is that the other oscillators work as expected. 

Is there something that get's saved outside of the Sound library location (which is why re-installing doesn't work?). 

I've made a screen recording to show the difference between the Sine and Sqr. 

https://dl.dropboxusercontent.com/u/1147392/sqrwave_bug.mov

Tested with Procesing 3.0b6 and 3.0b7. 
The library is 1.3.

**Comment by wirsing** [comment]
Hi Kasper,

the loundness of a square wave and a sine wave are obviously different because the overall sound level energy is higher in a square wave. The sine wave is one frequency and a square wave has a lot of overtones. Your video seems correct to me, the square wave is pretty loud in that video. Maybe I misunderstand your question?

all the best,
Wilm

**Comment by kasperkamperman** [comment]
I just picked the sine wave to compare, but the square wave has strange behaviour compared with all the wave. I hope you notice that in the video. With the volume on 0.5 it is barely hearable. To problem is that the square wave is a lot softer in sound. 

This happened suddenly. I work with a lot of students with the library and never had this behaviour. So I'm really curious what caused this.

**Comment by wirsing** [comment]
Mmh, i'm really sorry but I don't think I understand the problem. In the video the square wave is a lot louder than the sine wave, no? Can you maybe post the exact code where this is happening? Thanks!!

**Comment by kasperkamperman** [comment]
Strange indeed, in the video it is not visible. To be honest, I made the recording and probably I didn't watched it back at posting. The audio in the video was done through SoundFlower to record it directly. So what I encountered was maybe related to the direct audio output on the mac. What I said it was a strange error. I tried it right now again and it functions normally. 

Well, I'll close it for now, because I don't have any clues how to reproduce it either.
